pkgs/by-name/lp/lpac/package.nix

```diff
@@ -0,0 +1,60 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  lib,
+  cmake,
+  pkg-config,
+  pcsclite,
+  curl,
+  drivers ? true,
+  libeuicc ? true,
+  nix-update-script,
+}:
+
+let
+  inherit (lib) optional;
+in
+stdenv.mkDerivation (finalAttrs: {
+
+  pname = "lpac";
+  version = "2.2.1";
+
+  src = fetchFromGitHub {
+    owner = "estkme-group";
+    repo = "lpac";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-dxoYuX3dNj4piXQBqU4w1ICeyOGid35c+6ZITQiN6wA=";
+  };
+
+  env.LPAC_VERSION = finalAttrs.version;
+
+  patches = [ ./lpac-version.patch ];
+
+  cmakeFlags =
+    optional drivers "-DLPAC_DYNAMIC_DRIVERS=on"
+    ++ optional libeuicc "-DLPAC_DYNAMIC_LIBEUICC=on";
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
+
+  buildInputs = [
+    curl
+    pcsclite
+  ];
+
+  passthru = {
+    updateScriptEnabled = true;
```

Comment:
The`updateScriptEnabled` option is not used in upstream nixpkgs, it should be removed